src/main/kotlin/app/cash/licensee/plugin.kt

```diff
@@ -162,6 +166,26 @@ private fun configureAndroidVariants(
     rootTask.configure {
       it.dependsOn(task)
     }
+
+    if (extension.enableAndroidAssetGeneration.get()) {
+      val capitalizedVariantName = variant.name.replaceFirstChar { it.titlecase(ROOT) }
+      val copyArtifactsTask =
+        project.tasks.register(
+          "copy${capitalizedVariantName}LicenseeReportToAssets",
+          AssetCopyTask::class.java,
+        ) { asset ->
+          asset.targetFileName.set("artifacts.json")
```

Comment:
This should probably go into `app/cash/licensee/artifacts.json` rather than at the root to ensure we never collide with any other plugins or user data.

Reply:
thats a good point

Reply:
updated it here: https://github.com/cashapp/licensee/pull/412/files#diff-c1adba075e5541e1392a32c0e7731636bcbcac54eec1957b7e5676cfe6c5ffdcR47